### PR TITLE
Change spree docs link in README

### PR DESCRIPTION
**Pull request by arudnicka**
With actual docs, direct links don't move to the proper section.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -54,14 +54,14 @@ Node module for integration with Spree API V2
 
 1. Install SDK
 
-  `npm install spree-storefront-api-v2-js-sdk --save`
+  `npm install storefront-api-v2-sdk --save`
```
Comment (damianlegawiec) [inline README.md]: @arudnicka I think this should be `@spree/storefront-api-v2-sdk`

File: README.md [hunk 2]
```diff
@@ -54,14 +54,14 @@ Node module for integration with Spree API V2
...
 
 <br/>
 
 2. Initialize `Client`
 
   ```ts
-    import { spreeClient } from 'spree-storefront-api-v2-js-sdk'
+    import { spreeClient } from 'storefront-api-v2-sdk'
```
Comment (damianlegawiec) [inline README.md]: @arudnicka and also `@spree/` prefix here also

File: README.md [hunk 3]
```diff
@@ -178,13 +178,13 @@ Node module for integration with Spree API V2
...
 
-Spree Storefront API SDK contains each endpoint according to [Spree Guides](https://guides2.spreecommerce.org/api/v2)
+Spree Storefront API SDK contains each endpoint according to [Spree Guides](https://guides.spreecommerce.org/api/v2/storefront/)
 
 ## Authentication
 
 ### `getToken`
 
-Method `getToken` used for creates a Bearer token required to authorize calls API calls. [Read more](https://guides2.spreecommerce.org/api/v2/authentication)
+Method `getToken` used for creates a Bearer token required to authorize calls API calls. [Read more](https://guides.spreecommerce.org/api/v2/storefront//authentication)
```
Comment (tniezg) [inline README.md]: wrong link to `authentication`, should be `https://guides.spreecommerce.org/api/v2/storefront/#section/Authentication`

File: README.md [hunk 4]
```diff
@@ -224,7 +224,7 @@ __Example:__
 
 ### `refreshToken`
```
Comment (tniezg) [inline README.md]: Can't find `refreshToken` in Spree docs

**Comment by tniezg** [comment]
Most links don't work :/

File: README.md [hunk 5]
```diff
@@ -178,13 +178,13 @@ Node module for integration with Spree API V2
...
-Spree Storefront API SDK contains each endpoint according to [Spree Guides](https://guides2.spreecommerce.org/api/v2)
+Spree Storefront API SDK contains each endpoint according to [Spree Guides](https://guides.spreecommerce.org/api/v2/storefront)
 
-## Authentication
+## [Authentication](https://guides.spreecommerce.org/api/v2/storefront/#section/Authentication)
 
 ### `getToken`
 
-Method `getToken` used for creates a Bearer token required to authorize calls API calls. [Read more](https://guides2.spreecommerce.org/api/v2/authentication)
+Method `getToken` used for creates a Bearer token required to authorize calls API calls.
```
Comment (tniezg) [inline README.md]:
```suggestion
Method `getToken` creates a Bearer token required to authorize API calls.
```

File: README.md [hunk 6]
```diff
@@ -224,7 +224,7 @@ __Example:__
 
 ### `refreshToken`
 
-Method `refreshToken` used for refreshes a Bearer token required to authorize calls API calls. [Read more](https://guides2.spreecommerce.org/api/v2/authentication)
+Method `refreshToken` used for refreshes a Bearer token required to authorize calls API calls.
```
Comment (tniezg) [inline README.md]:
```suggestion
Method `refreshToken` refreshes a Bearer token required to authorize API calls.
```

**Pull request merged**